runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoOverlay.java

```diff
@@ -237,5 +268,17 @@ private void renderZalcanoAOE(Graphics2D graphics, int polySize, String text, Co
 		}
 	}
 
+	private void renderPlayerCount(Graphics2D graphics)
+	{
+		Rectangle rect = new Rectangle(client.getCanvasWidth() / 2 - 15, 0, 40, 25);
+		int[] xpoints = {rect.x, rect.x + rect.width, rect.x + rect.width, rect.x};
+		int[] ypoints = {rect.y, rect.y, rect.y + rect.height, rect.y + rect.height};
+
+		graphics.setFont(new Font("Comic Sans", Font.BOLD, 20));
```

Comment:
This should all be done how the rest of the client does overlay/ui. Not saying there's something inherently wrong with comic sans (well), but even a shitty but consistent ui looks better than a patchwork of beautiful ui's. Think rainbow graceful

Reply:
But it has to have big font to see it properly.

Reply:
I am not sure how to modify default font bigger.

Reply:
That minimap rock dots is ok. but player counter... well i have no idea how to make it nicer.